ambry-store/src/main/java/com.github.ambry.store/StoreFindToken.java

```diff
@@ -31,17 +34,12 @@
  * always equal or larger than the writable segment.
  */
 public class StoreFindToken implements FindToken {
-  /**
-   * The type of the store token.
-   */
-  public enum Type {
-    Uninitialized, JournalBased, IndexBased
-  }
 
   static final short VERSION_0 = 0;
   static final short VERSION_1 = 1;
   static final short VERSION_2 = 2;
-  static final short CURRENT_VERSION = VERSION_2;
+  static final short VERSION_3 = 3;
```

Comment:
Does version need to be kept in sync between the two sibling classes?  If so, that needs to be documented, but it also seems unclean.

Reply:
Might not be necessary. I will close if after discussing the versioning for CloudFindToken with you tomorrow.